src/python/grpcio/grpc/_cython/_cygrpc/aio/iomgr/timer.pyx.pxi

```diff
@@ -16,30 +16,32 @@
 cdef class _AsyncioTimer:
     def __cinit__(self):
         self._grpc_timer = NULL
-        self._timer_handler = None
-        self._active = 0
+        self._timer_future = None
+        self._active = False
+        cpython.Py_INCREF(self)
```

Comment:
Yes. In Line 34, and Line 47, the `_AsyncioTimer` is "decref"ed. This is the same pattern we used for `CallbackWrapper` that we manually modify refcount to simulate the behavior that Core is holding its reference. Otherwise, it will segfault.